src/Log/Backend/ElasticSearch.hs

```diff
@@ -146,29 +163,31 @@ elasticSearchLogger ElasticSearchConfig{..} genRandomWord = do
         Object jMsg = toJSON msg
 
     mkDocId :: BS.ByteString -> Word32 -> DocId
-    mkDocId baseID insertionOrder = DocId . T.decodeUtf8 . B64.encode $ BS.concat [
+    mkDocId baseID insertionOrder = DocId . T.decodeUtf8
+                                    . B64.encode $ BS.concat [
         baseID
       , littleEndianRep insertionOrder
       ]
 
-    toBulk :: IndexName -> BS.ByteString -> (Word32, H.HashMap T.Text Value) -> BulkOperation
-    toBulk index baseID (n, obj) = BulkIndex index mapping (mkDocId baseID n) $ Object obj
+    toBulk :: IndexName -> BS.ByteString -> (Word32, H.HashMap T.Text Value)
+           -> BulkOperation
+    toBulk index baseID (n, obj) =
+      BulkIndex index mapping (mkDocId baseID n) $ Object obj
 
 data LogsMapping = LogsMapping
 instance ToJSON LogsMapping where
   toJSON LogsMapping = object [
-      "logs" .= object [
-        "properties" .= object [
-          "insertion_order" .= object [
+    "properties" .= object [
+        "insertion_order" .= object [
             "type" .= ("integer"::T.Text)
           ]
         , "insertion_time" .= object [
             "type"   .= ("date"::T.Text)
-          , "format" .= ("strict_date_time"::T.Text)
+          , "format" .= ("date_time"::T.Text)
```

Comment:
We could go with 2.x on AWS, since we haven't set anything up yet, and I have no idea how hard it is to migrate, but it sounds like something to avoid.

@arybczak reported no problems with 2.3.3 so maybe we should try that... especially if `bloodhound` support is in the pipeline.